docs/src/pages/layout/jss.md

```diff
@@ -0,0 +1,8 @@
+# JSS
```

Comment:
What about calling this section CSS-in-JS?
I would also add a link to this section https://material-ui-1dab0.firebaseapp.com/customization/css-in-js regarding learning more about the styling solution.